sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -2817,4 +2817,42 @@ abstract class DDLSuite extends QueryTest with SQLTestUtils {
       }
     }
   }
+
+  test("SPARK-29421: Create Table LIKE USING provider") {
```

Comment:
Since this is not a bug fix, you don't need the prefix: `SPARK-29421: `.

Reply:
ok, thanks